ops/testing.py

```diff
@@ -15,10 +15,11 @@
 import inspect
 import pathlib
 import random
-from textwrap import dedent
 import tempfile
 import typing
 import yaml
+from contextlib import contextmanager
+from textwrap import dedent
```

Comment:
I'm not clear on pep8 here, are 'from' supposed to be sorted separately (I suppose if you're just 'sort' on the block this is the behavior you get.)

I mostly just want us to be consistent so we don't end up with 2 people editor's fighting over imports. :)

Reply:
This may be useful

https://github.com/PyCQA/isort

Reply:
PEP 8 doesn't explicitly say anything about where to put the "from" imports, but examples shown there follow this pattern of grouping all "from" imports at the end of the imports block.